ietf/meeting/urls.py

```diff
@@ -33,16 +33,8 @@
     url(r'^agenda/%(owner)s/%(schedule_name)s/delete$' % settings.URL_REGEXPS, views.delete_schedule),
     url(r'^agenda/%(owner)s/%(schedule_name)s/make_official$' % settings.URL_REGEXPS, views.make_schedule_official),
     url(r'^agenda/%(owner)s/%(schedule_name)s(\.(?P<ext>.html))?/?$' % settings.URL_REGEXPS, views.agenda),
-    url(r'^agenda/%(owner)s/%(schedule_name)s/week-view(?:.html)?/?$' % settings.URL_REGEXPS, views.week_view),
```

Comment:
I expect removal of this will bring a request from someone later to provide a rendered calendar view unencumbered by other dressing. But we can wait for that request before we re-build that functionality.

Reply:
(This particular variant of the URI gets very low use, but the one that defaults to the official agenda below, I suspect, gets more.